src/gsy_e/models/strategy/external_strategies/__init__.py

```diff
@@ -349,11 +349,10 @@ def _offer_aggregator_impl(
 
             offer_arguments = {k: v
                                for k, v in arguments.items()
-                               if k not in ["transaction_id", "type", "time_slot"]}
+                               if k not in ["transaction_id", "type"]}
```

Comment:
I think that it would be better to pick one of the two options:
1. We keep the code in gsy-e, so we also maintain it and keep it working
2. We do not maintain this code, so we remove it completely.

I think that if we keep a code that is "wrong" and doesn't work, this won't help in the future and it will actually confuse readers. Even just the fact that the code is here would raise the question "so are we supporting this or not?". In my opinion, unused code should (almost) always be deleted. If we want to put it back where it was, we can check the git history (given we use proper commit messages).

What do you think?

Reply:
We had multiple long discussions about this before and I remember us deciding to keep it. The code can still be used, just not with the sdk. If you write your own client, you could still post direct orders via the gsy-web or via a redis connection. Only our sdk is not exposing this possibility. 
So, let me go for the first option and add the possibility to post on the future market. I am not sure, but I think that also the settlement market is not supported there. So, this might be even worth a separate story. 
Let me start and if I think it is too much, I will create a ticket (could also be an onboarding task)